### PR TITLE
netdev bond: add support to configure tlb_dynamic_lb

### PR DESCRIPTION
**Pull request by ssahani**
Closes https://github.com/systemd/systemd/issues/11135

### CODE REVIEW

**Comment by ssahani** [comment]
> I am not familiar with this option, but why do not you use 'tristate' for this option?

very much true

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -1457,6 +1457,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TlbDynamicLb=</varname></term>
```
Comment (poettering) [inline man/systemd.netdev.xml]: urks, what does "Tlb" and "Lb" stand for? I am left very confused about the naming of this option

File: src/network/netdev/netdev-gperf.gperf [hunk 2]
```diff
@@ -140,7 +140,8 @@ Bond.PrimaryReselectPolicy,        config_parse_bond_primary_reselect,   0,
 Bond.ResendIGMP,                   config_parse_unsigned,                0,                             offsetof(Bond, resend_igmp)
 Bond.PacketsPerSlave,              config_parse_unsigned,                0,                             offsetof(Bond, packets_per_slave)
 Bond.GratuitousARP,                config_parse_unsigned,                0,                             offsetof(Bond, num_grat_arp)
-Bond.AllSlavesActive,              config_parse_unsigned,                0,                             offsetof(Bond, all_slaves_active)
+Bond.AllSlavesActive,              config_parse_bool,                    0,                             offsetof(Bond, all_slaves_active)
```
Comment (poettering) [inline src/network/netdev/netdev-gperf.gperf:143]: hmm, this looks like an indepdendent bugfix. could you split this into a separate commit please (in the same PR, that'd be fine)

File: man/systemd.netdev.xml (hunk 1 above)
Comment (ssahani) [reply man/systemd.netdev.xml]: :)  LB-> Load Balancing
    T->Transmit

**Comment by ssahani** [comment]
updated thanks for review .

File: man/systemd.netdev.xml (hunk 1 above)
Comment (poettering) [reply man/systemd.netdev.xml]: hmm, the expanded form of this option would be TransmitLoadBalancingDynamicLoadBalancing? i.e. LoadBalancing appears twice?

How about calling this DynamicTransmitLoadBalancing=?

Comment (ssahani) [reply man/systemd.netdev.xml]: yes nice.

**Comment by evverx** [comment]
Travis CI failed due to https://github.com/systemd/systemd/issues/11170.
